src/EluvioLive.js

```diff
@@ -43,8 +43,12 @@ class EluvioLive {
     this.client.ToggleLogging(false);
   }
 
-  async InitNew() {
-    this.client = await ElvClient.FromConfigurationUrl({
+  async CreateAccount({ funds = 0.1, accountName, tenantId }) {
```

Comment:
Client and core consider < 0.1 eth to be effectively 'out of funds'. Recommend bumping this to at least 0.25, if not 1.

Reply:
Bumped to .25, thanks.